pkgs/tools/security/cpu_rec/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, python3Packages, fetchFromGitHub }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "cpu_rec";
+  version = "unstable-2021-06-21";
```

Comment:
```suggestion
  version = "2021-06-21";
```

Reply:
This is my fault. I didn't notice that the manual has been changed. Please ignore it.